pkg/virtctl/imageupload/imageupload.go

```diff
@@ -612,11 +636,19 @@ func createStorageSpec(client kubecli.KubevirtClient, size, storageClass, access
 	return spec, nil
 }
 
-func createUploadPVC(client kubernetes.Interface, namespace, name, size, storageClass, accessMode string, blockVolume, archiveUpload bool) (*v1.PersistentVolumeClaim, error) {
+func createUploadPVC(client kubecli.KubevirtClient, namespace, name, size, storageClass, accessMode string, blockVolume, archiveUpload bool) (*v1.PersistentVolumeClaim, error) {
 	if accessMode == string(v1.ReadOnlyMany) {
 		return nil, fmt.Errorf("cannot upload to a readonly volume, use either ReadWriteOnce or ReadWriteMany if supported")
 	}
 
+	// We check if the storageClass exists before attempting to create the PVC
```

Comment:
We explicitly didn't check for storage class as it is entirely valid to have a kubernetes cluster without a storage class, but with manually provisioned PVs. minikube or CRC are good examples of this. This change would make upload no longer work on those.

Reply:
Same as above.